MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: post
+title: Header and footer with .NET MAUI Picker control | Syncfusion
+description: Learn here all about header and footer with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Dealing with Header and Footer in .NET MAUI Picker (SfPicker)
+
+This section explains about the header and footer customization of picker control.
```

Comment:
This section explains the header and footer customization of picker control.

Reply:
Content modified